src/ui/pages/SendTransaction/TransactionAdvancedView/TransactionAdvancedView.tsx

```diff
@@ -74,21 +74,19 @@ function AddressLine({
 }
 
 function TransactionDetails({
-  networks,
-  chain,
+  network,
   transaction,
-  interpretation,
-}: {
-  networks: Networks;
-  chain: Chain;
+}: // interpretation,
+{
+  network: NetworkConfig;
   transaction: IncomingTransaction;
   interpretation?: InterpretResponse | null;
 }) {
-  const functionName = useMemo(
-    () =>
-      interpretation ? getInterpretationFunctionName(interpretation) : null,
-    [interpretation]
-  );
+  // const functionName = useMemo(
+  //   () =>
+  //     interpretation ? getInterpretationFunctionName(interpretation) : null,
+  //   [interpretation]
+  // );
```

Comment:
rm comments in this file? Is this the final version?

Reply:
Yes, I will remove comments!